src/service/node/slave.cpp

```diff
@@ -27,13 +27,14 @@
 #include "cocaine/detail/service/node/engine.hpp"
 #include "cocaine/detail/service/node/event.hpp"
 #include "cocaine/detail/service/node/manifest.hpp"
-#include "cocaine/detail/service/node/messages.hpp"
 #include "cocaine/detail/service/node/profile.hpp"
 #include "cocaine/detail/service/node/session.hpp"
 #include "cocaine/detail/service/node/stream.hpp"
 
 #include "cocaine/logging.hpp"
 
+#include "cocaine/idl/rpc.hpp"
```

Comment:
These are usually alphabetic throughout the code, but it's your call in Node Service.